pkg/cmd/codespace/create.go

```diff
@@ -153,7 +152,13 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		}
 	}
 
+	cs := a.io.ColorScheme()
+
 	fmt.Fprintln(a.io.Out, codespace.Name)
+	if codespace.IdleTimeoutNotice != "" {
+		fmt.Fprintln(a.io.Out, cs.Cyan("Notice:"), codespace.IdleTimeoutNotice)
```

Comment:
The contract of `gh cs create` method is that it prints the identifier of the just-created codespace to stdout so that it can be read by scripts. If we print anything else to stdout, like this notice, we break that contract and `gh cs create` is suddenly unusable in scripting.

I would print to stderr instead and only if stderr is connected to a terminal, ensuring that a human would see the message:
```go
if a.io.IsStderrTTY() { fmt.Fprintln(a.io.ErrOut, ...
```
Finally, I think yellow might be a better color for the notice than cyan since we also use error to print non-fatal warnings.